metadata/src/test/java/edu/unc/lib/dl/util/RDFModelUtilTest.java

```diff
@@ -1,98 +0,0 @@
-/**
```

Comment:
is RDFModelUtil being retired completely? i noticed you left some methods in the class, but wasn't sure whether these were just to avoid compile errors for the time being

Reply:
Its not being totally retired, I just readjusted the scope of it so that it is just about serializing, deserializing and streaming models.  Creating sparql query strings seemed pretty distinct from those other actions.